cmd/copyEnumeratorInit.go

```diff
@@ -171,8 +183,9 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 					// this will probably never be reached
 					return nil, fmt.Errorf("failed to get container name from source (is it formatted correctly?)")
 				}
+				var resName string
 
```

Comment:
Was there any logical impact of this change?

Reply:
Discarded refactor. Edited it back.
https://github.com/Azure/azure-storage-azcopy/pull/1036#discussion_r436445994